src/Database/LSMTree/Internal/Monkey.hs

```diff
@@ -37,8 +39,8 @@ monkeyFPR numBits numEntries =
 monkeyBits ::
      Int    -- ^ Number of entries inserted into the bloom filter.
   -> Double -- ^ False positive rate.
-  -> Int
-monkeyBits numEntries fpr = ceiling $
+  -> Word64
+monkeyBits numEntries fpr = max 1 $ ceiling $
   (- fromIntegral numEntries) * (log fpr / (log 2 ** 2))
```

Comment:
This leads to underflow for FPR > 1, which shouldn't happen, but maybe best to add a guard or at least switch `max` and `ceiling`?

Reply:
Good catch -- I've switched them around